src/main/java/uk/gov/justice/hmpps/prison/api/resource/AgencyResource.java

```diff
@@ -100,8 +99,7 @@ public Agency getAgency(@PathVariable("agencyId") @ApiParam(value = "The ID of t
         @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class)})
     @ApiOperation(value = "Update an existing agency", notes = "Requires MAINTAIN_REF_DATA")
     @PutMapping("/{agencyId}")
-    @HasWriteScope
-    @PreAuthorize("hasRole('MAINTAIN_REF_DATA')")
+    @PreAuthorize("hasRole('MAINTAIN_REF_DATA') and hasAuthority('SCOPE_write')")
```

Comment:
Can't have two PreAuthorize on same method so have combined for now and remove HasWriteScope

Reply:
Good find - that was my bad I think!